src/app/test_executive/zkapps.ml

```diff
@@ -749,7 +749,7 @@ module Make (Inputs : Intf.Test.Inputs_intf) = struct
       section_hard "Send a zkapp with an invalid proof"
         (send_invalid_zkapp ~logger
            (Network.Node.get_ingress_uri node)
-           zkapp_command_invalid_proof "Verification_failed" )
+           zkapp_command_invalid_proof "invalid" )
```

Comment:
Same here, This would be `Invalid_proof` (going by the description)

Reply:
Replied in previous case.